portfolio/src/main/java/com/google/sps/data/EntityProperties.java

```diff
@@ -5,15 +5,30 @@
 
   private EntityProperties() {}
 
-  /* Entity Properties: */
+  /* Comment Properties: */
 
   /** The property representing the text of a comment on the webpage. */
   public static final String COMMENT_TEXT = "text";
 
-  /** The property representing the name of the author of a comment. */
-  public static final String COMMENT_AUTHOR = "name";
+  /** The property representing the user id of the author of a comment. */
+  public static final String USER_ID_OF_AUTHOR = "userId";
 
   /** The property representing the timestamp when the comment was posted (in 
     * milliseconds since the epoch). */
   public static final String COMMENT_TIMESTAMP = "time";
+
+  /** The property representing the numeric id of this entiity in the 
+    * datastore. Used for deleting comments. */
+  public static final String COMMENT_ID = "id";
+
+  /** The property representing the email of the author of this comment */
+  public static final String USER_EMAIL = "email";
+
+  /* User Properties */
+
+  /** The property representing the user id */
+  public static final String USER_ID = "userId";
```

Comment:
Isn't this the same as USER_ID_OF_AUTHOR?

Reply:
Yes, I was going back and forth on whether to define two separate constants. My reasoning was that they're used in to different kinds of entities that just happen to reference the same attribute, and I had structured my constants file to be organized by kind of entity. But, I can see that this is probably more confusing. I've changed it!